src/mame/drivers/ti85.cpp

```diff
@@ -1032,6 +1032,13 @@ ROM_START (ti84pv2)
 	ROMX_LOAD( "ti84pv2v255mp.bin", 0x00000, 0x100000, CRC(3f8574a6) SHA1(0f88e719512f2691fff6c8bcc89292158086f841), ROM_BIOS(8) )
 ROM_END
 
+ROM_START (ti84pv2)
+	ROM_REGION (0x100000, "flash",0)
+	ROM_DEFAULT_BIOS("v255mp")
+	ROM_SYSTEM_BIOS( 0, "v255mp", "V 2.55MP" )
+	ROMX_LOAD( "ti84pv2v255mp.bin", 0x00000, 0x100000, CRC(13d7d311) SHA1(c0fd22cd822d77dde0b037c33123178e30275d27), ROM_BIOS(1) )
```

Comment:
Looks to be another copy/paste sharing the label of the previous romset you copied from.